resources/dicts/events/death/general/general.json

```diff
@@ -304,7 +304,7 @@
     {
         "tags": ["other_cat", "Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare"],
         "camp": "Any",
-        "death_text": "m_c has always wondered where {PRONOUN/m_c/subject} came from. {PRONOUN/m_c/subject/CAP} {VERB/sneak/sneaks} out of camp, intending to find loners who might have some clues about {PRONOUN/m_c/poss} origins, when suddenly {PRONOUN/m_c/subject} {VERB/is/are} ambushed and killed by a badger. c_n finds {PRONOUN/m_c/poss} body after a long search and wonders why {PRONOUN/m_c/subject} decided to leave the safety of the c_n.",
+        "death_text": "m_c has always wondered where {PRONOUN/m_c/subject} came from. {PRONOUN/m_c/subject/CAP} {VERB/sneak/sneaks} out of camp, intending to find loners who might have some clues about {PRONOUN/m_c/poss} origins. Frantic with worry and confusion after finding m_c's empty nest in the morning, they launch a search. c_n finds {PRONOUN/m_c/poss} cooling body after a long search and wonders why {PRONOUN/m_c/subject} decided to leave the safety of the c_n, and what killed {PRONOUN/m_c/object}.",
         "history_text": {
             "reg_death": "Curiosity killed this cat.",
             "lead_death": "lost a life because of curiosity."
```

Comment:
forgot to mention this before, but if you want a leader to be able to get this then you'll have to copy paste it into the leader.json as well.  leaders don't have access to the general death events.  This also applies to the elder old age event.